### DISCUSSION TITLE
MERN - Course-end Project

### DISCUSSION THREAD

**Discussion by akash-coded** [akash-coded/mern#89] [Tasks]
Create the project according to the specifications given in the project documentation.

**Comment by santu1667** [comment]
@akash-coded  Hey Akash , Please check and review the attached folder so i will share the same for final submission
https://drive.google.com/drive/folders/1CHGj3EwrTlZqC2Wi_XD0eddhAbVABye6?usp=sharing

**Comment by skagarwal5555** [comment]
[Fullstack Training Certification project.docx](https://github.com/akash-coded/mern/files/9292889/Fullstack.Training.Certification.project.docx)

Hi Akash, Please review the above document. Also here is the github link
https://github.com/skragarwaltcs/mern/tree/master/CERTIFICATION%20PROJECT

I have additionally implemented the feature for Add/Edit/Delete users for an admin user which were features mentioned in the project requirement but no sample UI or api was mentioned.

**Comment by uday2305** [comment]
[Shop24x7_Project.pdf](https://github.com/akash-coded/mern/files/9355028/Shop24x7_Project.pdf)

Hi Akash,

Please find the attached document, which contains all the screen-shots of the guest, logged-in user, Admin User screen flows.

Please find the code repo link : [here](https://github.com/uday2305/learning_mern)

Please review and share your feedback.

**Comment by sena-83** [comment]
Hey Akash, 
Please find the attached document for your review -
[Fullstack_Training_Certification_Project_Doc.pdf](https://github.com/akash-coded/mern/files/9357400/Fullstack_Training_Certification_Project_Doc.pdf)

You are able to see frontend and backend code from here (latest code been checked in here) -
(https://github.com/sena-83/MERN)

As per your suggestion in last session, I have built a modal component and that component been leveraged in multiple places for showing confirmation messages and alerts. Category based and price range-based search also been included as discussed in the last session.

As per latest direction from SPOC, we have to submit final project into LMS by 17th Aug, that's why I have submitted the frontend and backend code in LMS along with this detail document and mongo db collections export, postman collection export too for your reference.

**Comment by MRamitkv** [comment]
Hi Akash,

I have uplaoded the code with DB export at this location:
https://drive.google.com/drive/folders/1du2SbD6YI27kDuhyL9S_6ftbqR1kVKb5?usp=sharing

I have also uploaded the screenshot.
the bootstrap carousal was not working in my system. So I implemented similar functionality in css and javascript.
